Dockerfile

```diff
@@ -28,25 +28,24 @@ RUN ./install_cuda.sh && \
     rm install_cuda.sh
 
 # System dependencies
-RUN apt update && apt install -y wget git python3 python3-pip zip
+RUN apt update && apt install -y wget git python3 python3-pip zip libgl1 libgl1-mesa-glx libglib2.0-0
```

Comment:
Would be useful to have a comment explaining what requires the libgl stuff.

Reply:
@AIWintermuteAI can you give me a comment to add to the dockerfile or add one in?

Reply:
Sure. Without these opencv cannot be imported. 
https://stackoverflow.com/questions/55313610/importerror-libgl-so-1-cannot-open-shared-object-file-no-such-file-or-directo
I tried a few different packages, I think I got the correct minimal set here.

Reply:
It was building, but would throw an error at
import cv2